docs/platform_ccl.rst

```diff
@@ -0,0 +1,458 @@
+.. _cclplatform:
+
+CC-Light Platform Configuration File
+------------------------------------

```

Comment:
This seems a copy from the old file in the "doc" folder, let's remove the old one.